### PR TITLE
chore: add test to logger.error on firebase initialization

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- add test to logger.error on firebase initialization in order to increase coverage

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Comment by luislhl** [comment]
I think we have a problem on `https://github.com/HathorNetwork/hathor-wallet-service/blob/master/.github/workflows/pr-validation.yml`

You can see that in this PR, the tests didn't run yet.

This is because the config in the workflow is preventing them from running for the first commit of each PR.

We should create another PR to fix this.

The fix is to remove these lines:
```
    types: [ready_for_review, synchronize]
    branches: [master, dev]
```

The `types` config is what causes the problem. We should just use default (by removing this line).

The `branches` config, if kept, will prevent PRs that have other bases from running tests as well.
I think it doesn't make sense.

File: .github/workflows/pr-validation.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: PR Validation
 on:
   pull_request:
-    types: [ready_for_review, synchronize]
-    branches: [master, dev]
+    types: [opened, synchronize]
```
Comment (luislhl) [inline .github/workflows/pr-validation.yml]: The github actions default is `[opened, synchronize, reopened]`

Did you leave `reopened` out for a specific reason?
It seems that we should have it.

But I don't think this is a big deal, so I'll not block the PR for this

Comment (alexruzenhack) [reply .github/workflows/pr-validation.yml]: Oh, I didn't know about the default. I will ripe it off then.

File: tests/utils/pushnotification.utils.test.ts [hunk 2]
```diff
@@ -37,6 +37,25 @@ describe('PushNotificationUtils', () => {
...
   });
 
+  // test firebase initialization error
+  it('firebase initialization error', () => {
+    expect.hasAssertions();
+
+    // load local env
+    process.env.PUSH_NOTIFICATION_ENABLED = 'true';
+    logger.error.mockReset();
+    initFirebaseAdminMock.mockReset();
```
Comment (luislhl) [inline tests/utils/pushnotification.utils.test.ts:47]: Maybe it's better to add these 2 `mockReset` calls to the `afterEach` function as well, so we make sure we won't be leaving traces that could impact other tests. 

What do you think?

Unless there is some other process in place somewhere to reset all mocks.

**Review by luislhl — APPROVED** [review]

File: tests/utils/pushnotification.utils.test.ts (hunk 2 above)
Comment (alexruzenhack) [reply tests/utils/pushnotification.utils.test.ts:47]: This is a good point, but I don't like to generalize with few applications, in this case only this test needs this reset.

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**